lib/get-selenium-status-url.js

```diff
@@ -68,7 +69,7 @@ exports.getSeleniumStatusUrl = function (seleniumArgs) {
   }
 
   const statusURI = new URL('http://' + host);
-  const nodeStatusAPIPath = '/wd/hub/status';
+  const nodeStatusAPIPath = defaultConfig.version.startsWith('4.') ? '/status' : '/wd/hub/status';
```

Comment:
Don't use default config to determine version

Reply:
using opts here